rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/IRascalFileSystemServices.java

```diff
@@ -73,13 +77,8 @@ default CompletableFuture<SourceLocation> resolveLocation(SourceLocation loc) {
                 }
 
                 return SourceLocation.fromRascalLocation(resolved);
-            }
-            catch (URISyntaxException e) {
-                IRascalFileSystemServices__logger.warn("Could not resolve location {} due to {}.", loc, e.getMessage());
-                return loc;
-            }
-            catch (Throwable e) {
-                IRascalFileSystemServices__logger.warn("Could not resolve location {} due to {}.", loc, e.getMessage());
+            } catch (Exception e) {
+                IRascalFileSystemServices__logger.warn("Could not resolve location {}", loc, e);
```

Comment:
Do we really need to catch all exception types here? That always runs the risk of masking programming errors.  Warnings are easily overlooked.

Reply:
Well, the resolve we generally don't want to crash, normally, yes, we forward all exceptions, but resolve shouldn't crash.